src/prefect/cli/task.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+import typer
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.task_server import serve as task_serve
+from prefect.utilities.importtools import import_object
+
+task_app = PrefectTyper(name="task", help="Commands for working with task scheduling.")
+app.add_typer(task_app, aliases=["task"])
+
+
+@task_app.command()
+async def serve(
+    entrypoints: List[str] = typer.Argument(
```

Comment:
just for my understanding, why a `List[str]` vs `List[Path]`?

Reply:
They're not paths because we're expecting them to have `:` in them.

Reply:
oh I see